controller/alb/loadbalancers.go

```diff
@@ -28,6 +30,12 @@ func (l LoadBalancers) Reconcile() (LoadBalancers, LoadBalancers) {
 			errLBs = append(errLBs, loadbalancer)
 			continue
 		}
+		if loadbalancer.ResourceRecordSet == nil {
```

Comment:
Perhaps this check is too late.

If `ResourceRecordSet` is nil, can we prevent `Reconcile` from occurring in the first place?

Specifically, could we do a similar check somewhere around here: https://github.com/coreos/alb-ingress-controller/blob/master/controller/ingress.go#L164-L174.